config/pitchfork.conf.rb

```diff
@@ -20,7 +20,7 @@
 worker_processes (ENV["UNICORN_WORKERS"] || 3).to_i
 
 # stree-ignore
-listen ENV["UNICORN_LISTENER"] || "#{(ENV["UNICORN_BIND_ALL"] ? "" : "127.0.0.1:")}#{(ENV["UNICORN_PORT"] || 3000).to_i}"
+listen ENV["UNICORN_LISTENER"] || "#{(ENV["UNICORN_BIND_ALL"] ? "" : "127.0.0.1:")}#{(ENV["UNICORN_PORT"] || 3000).to_i}", reuseport: true
```

Comment:
This option allows Pitchfork to start a new process even if the old one is still running. (they’re sharing their port)